apps/sage_intacct/models.py

```diff
@@ -500,12 +519,12 @@ def create_charge_card_transaction(expense_group: ExpenseGroup):
                 expense_group=expense_group,
                 defaults={
                     'charge_card_id': charge_card_id,
+                    'vendor_id': vendor,
                     'description': description,
-                    'memo': 'Reimbursable expenses by {0}'.format(description.get('employee_email')) if
-                    expense_group.fund_source == 'PERSONAL' else
-                    'Credit card expenses by {0}'.format(description.get('employee_email')),
+                    'memo': 'Credit card expenses by {0}'.format(description.get('employee_email')),
+                    'reference_no': expense.expense_number,
                     'currency': expense.currency,
-                    'transaction_date': get_transaction_date(expense_group)
+                    'transaction_date': expense_group.description['spent_at']
```

Comment:
CCT can only be created for CCC expenses

Reply:
Lets not change this, if spent at is null current date should be exported
```suggestion
                    'transaction_date': get_transaction_date(expense_group)
```